configure.ac

```diff
@@ -205,6 +205,43 @@ AC_RUN_IFELSE([
    ],
    [AC_MSG_RESULT(skipped)])
 
+AC_MSG_CHECKING(for __builtin_clz)
+AC_COMPILE_IFELSE([
+   AC_LANG_PROGRAM([], [__builtin_clz(1); /* Requires gcc 3.4 or later */])],
+   [AC_DEFINE([HAVE_BUILTIN_CLZ], 1, [Define to 1 if the compiler supports `__builtin_clz' function.])
+   AC_MSG_RESULT(yes)],
+   AC_MSG_RESULT(no))
+
+case "$host_cpu" in
+   arm* | aarch64*)
+      AC_MSG_CHECKING(for ARM NEON intrinsics)
+      AC_COMPILE_IFELSE([
+         AC_LANG_PROGRAM([[
+            #ifndef __ARM_NEON
+            choke me
```

Comment:
```suggestion
            #error __ARM_NEON not available
```

Reply:
The message is invisible to the builder. It can be seen only by editors of `configure.ac` script. And [Autoconf manual suggests not to use `#error` directive](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.70/html_node/C-Compiler.html#C-Compiler) in the test.

Reply:
But this message is visible in the `config.log` ;-) And for those compilers that don't report an error when encountering `#error` I'd kindly reference that [iconic NVidia quote by L. Torvalds](https://arstechnica.com/information-technology/2012/06/linus-torvalds-says-f-k-you-to-nvidia/). ;-)

Anyway this conditional is just a pre-check. If that pre-check fails it's very likely that the following `#include` will trigger a compilation failure due to missing files anyway. No harm done. ;-)

FWIW: IRIX has been retired about 8 years ago …

Reply:
It's not that the compiler won't report the error. It's about whether the compiler will halt by the `#error` directive. I have an idea  to incorporate both anyway…

Reply:
As said, this ain't a problem, because without that define present, it's more than likely that the following `#include` from that test will fail because of that file missing or some other compile issue. And even with that file those intrinsics will not be available. Thus overall that test program likely won't compile and THAT is what will flag that configure test as failed. The `#error` in that case is just a neat shortcut in cases where it's supported …

Reply:
I was overthinking. There is no need to check for `__ARM_NEON` at all in the configure test. The test is about whether NEON intrinsics are _supported_, and there's no need to care if they are actually _enabled_. And the compiler will fail if it couldn't recognize `vcreate_u8` anyway.